BaseStation/BaseStation.sln

```diff
@@ -26,49 +26,133 @@ EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
+		Debug|x64 = Debug|x64
```

Comment:
Do you know why visual studio modified this?

Reply:
I think because i was messing with some of the settings to fix a warning about mismatching processor stuff